### PR TITLE
Feature/UI

### PR DESCRIPTION
**Pull request by OlgaPinchuk**
Please review my PR

### CODE REVIEW

File: src/main/java/com/novare/inventoryManager/employees/Employee.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.security.NoSuchAlgorithmException;
+import java.util.UUID;
+
+public class Employee implements Serializable {
+    private UUID id;
+    private String fullName;
+    private String socialNumber;
+    private String employmentNumber;
```
Comment (AlirezaKToosi) [inline src/main/java/com/novare/inventoryManager/employees/Employee.java]: We have already removed this field from schema

File: src/main/java/com/novare/inventoryManager/menus/adminMenu/AdminMenuController.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        Employee newEmployee;
+
+        view.printAddNewEmployeeRequest(role);
+
+        while (true) {
+            String fullName = Menu.getInput(view.FULL_NAME_REQUEST);
+            String socialNumber = Menu.getInput(view.SOCIAL_NUMBER_REQUEST);
+            String employmentNumber = Menu.getInput(view.EMPLOYMENT_NUMBER_REQUEST);
+            BigDecimal salary = new BigDecimal(Menu.getInput(view.SALARY_REQUEST));
+
```
Comment (AlirezaKToosi) [inline src/main/java/com/novare/inventoryManager/menus/adminMenu/AdminMenuController.java:98]: Is the input value validated for salary?
For example, the number should not be negative

File: src/main/java/com/novare/inventoryManager/menus/adminMenu/AdminMenuController.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            throw new RuntimeException(e);
+        }
+    }
+
+    void handleOption(int selectedOption) throws IndexOutOfBoundsException, NoSuchAlgorithmException {
+        try {
+            switch (selectedOption) {
+                case 1 -> addNewEmployee(EmployeeRole.MANAGER);
+                case 2 -> addNewEmployee(EmployeeRole.CASHIER);
+
```
Comment (AlirezaKToosi) [inline src/main/java/com/novare/inventoryManager/menus/adminMenu/AdminMenuController.java:43]: Admin should be able to see the list of employee based on the UseCase diagram

File: src/main/java/com/novare/inventoryManager/menus/loginMenu/LoginMenuController.java [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    private void changePassword(Employee employee) throws NoSuchAlgorithmException {
+        view.printChangePasswordOption();
+        String newPassword = getCustomerPassword(view.NEW_PASSWORD_REQUEST);
+        if (newPassword == null) {
+            view.printPasswordError();
+            return;
+        }
+
+        String hashedPassword = PasswordHasher.hashPassword(newPassword);
+        employee.setPassword(hashedPassword);
```
Comment (AlirezaKToosi) [inline src/main/java/com/novare/inventoryManager/menus/loginMenu/LoginMenuController.java]: Verify the password is Strong or not (Password should be at least 8 characters and contain 1 uppercase, 1 lowercase, and 1 number)

File: src/main/java/com/novare/inventoryManager/menus/cashierMenu/CashierMenuController.java [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+    void handleOption(int selectedOption) throws IndexOutOfBoundsException {
+        try {
+            switch (selectedOption) {
+
+                case 1 -> System.out.println("TODO: View the inventory products");
+                case 2 -> System.out.println("TODO: Add orders to the orders list");
+                case 3 -> System.out.println("TODO: Sell an item");
+                case 4 -> System.out.println("TODO: Open the group chat");
+
```
Comment (AlirezaKToosi) [inline src/main/java/com/novare/inventoryManager/menus/cashierMenu/CashierMenuController.java:41]: Add change password in CashierMenu

File: src/main/java/com/novare/inventoryManager/menus/managerMenu/ManagerMenuController.java [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+    void handleOption(int selectedOption) throws IndexOutOfBoundsException {
+            switch (selectedOption) {
+                case 1 -> System.out.println("TODO: View the inventory products/entries");
+                case 2 -> System.out.println("TODO: Generate report");
+                case 3 -> System.out.println("TODO:  Add new product");
+                case 4 -> System.out.println("TODO: Add orders to the orders list");
+                case 5 -> System.out.println("TODO: Add receipts");
+                case 6 -> System.out.println("TODO: Open the group chat");
+
```
Comment (AlirezaKToosi) [inline src/main/java/com/novare/inventoryManager/menus/managerMenu/ManagerMenuController.java]: Add change password in ManagerMenu

**Review by AlirezaKToosi — CHANGES_REQUESTED** [review]
Thanks , I have sent the changes that seemed to me. In addition, I think the test part is also needed.

File: src/main/java/com/novare/inventoryManager/employees/Employee.java (hunk 1 above)
Comment (OlgaPinchuk) [reply src/main/java/com/novare/inventoryManager/employees/Employee.java]: I removed it

File: src/main/java/com/novare/inventoryManager/menus/adminMenu/AdminMenuController.java (hunk 2 above)
Comment (OlgaPinchuk) [reply src/main/java/com/novare/inventoryManager/menus/adminMenu/AdminMenuController.java:98]: I will add validation to the final version

File: src/main/java/com/novare/inventoryManager/menus/adminMenu/AdminMenuController.java (hunk 3 above)
Comment (OlgaPinchuk) [reply src/main/java/com/novare/inventoryManager/menus/adminMenu/AdminMenuController.java:43]: to do

File: src/main/java/com/novare/inventoryManager/menus/loginMenu/LoginMenuController.java (hunk 4 above)
Comment (OlgaPinchuk) [reply src/main/java/com/novare/inventoryManager/menus/loginMenu/LoginMenuController.java]: done

File: src/main/java/com/novare/inventoryManager/menus/cashierMenu/CashierMenuController.java (hunk 5 above)
Comment (OlgaPinchuk) [reply src/main/java/com/novare/inventoryManager/menus/cashierMenu/CashierMenuController.java:41]: Move to the tech debt. Will be done later during this sprint or moved to the next

File: src/main/java/com/novare/inventoryManager/menus/managerMenu/ManagerMenuController.java (hunk 6 above)
Comment (OlgaPinchuk) [reply src/main/java/com/novare/inventoryManager/menus/managerMenu/ManagerMenuController.java]: Move to the tech debt. Will be done later during this sprint or moved to the next

**Comment by aggarrohit** [comment]
Before showing user menus, I thing that screen is cleared, but when we will actually the inventory or show any other info then screen should not be cleared otherwise the printed info will also be cleared. 
However I am not sure as I am not running the app in terminal yet, so please check once.

**Review by AlirezaKToosi — APPROVED** [review]

**Pull request merged**